spec/README.md

```diff
@@ -123,3 +123,28 @@ All other names in these categories are reserved for the use of implementations
 > [!NOTE]
 > Users defining custom names SHOULD include at least one character outside these ranges
 > to ensure that they will be compatible with future versions of this specification.
+
+Later versions of this specification will not introduce changes
+to the data model that would result in a data model representation
+based on this version being invalid.
+
+> For example, existing interfaces or fields will not be removed.
```

Comment:
What about changes to the existing fields and interfaces?

IIUC, this parts says that if we every un-reserve, say, the `%` sigil, we will also need to keep it on the list of sigils inside `UnsupportedAnnotation`:

```
interface UnsupportedAnnotation {
  type: "unsupported-annotation";
  sigil: "!" | "%" | "^" | "&" | "*" | "+" | "<" | ">" | "?" | "~";
  source: string;
}
```

Otherwise, (i.e. if we remove `%` from the list), data model representations produced by older implementations will not be valid.

---

What was the reason for including the `sigil` field in `UnsupportedAnnotation`? Could we move the sigil into the `source`?

Reply:
Good point. Including it in the `source` string would be a better choice for future compatibility.